python/lsst/ctrl/ingestd/mapper.py

```diff
@@ -36,17 +36,17 @@ class Mapper:
     def __init__(self, rse_dict: dict):
```

Comment:
It might be good for future maintenance to adjust the class doc, the parameter name, and the internal variable name to use `topic` rather than `rse`.

Reply:
I changed it.